tests/avalon_test_framework/src/libs/submit_request.py

```diff
@@ -21,41 +21,45 @@
 from avalon_sdk.connector.blockchains.ethereum.ethereum_work_order \
     import EthereumWorkOrderProxyImpl
 import avalon_sdk.worker.worker_details as worker_details
+from conftest import env
+
 logger = logging.getLogger(__name__)
 
+tcf_connector_conffile = [env['tcf_connector_conffile']]
```

Comment:
Move the 2 lines below inside the config_read_file function only

Reply:
Changes addressed